### DISCUSSION TITLE
SIEM logs for Teleport Cloud

### DISCUSSION THREAD

**Discussion by gord-lyr** [gravitational/teleport#20493] [Q&A] [answered]
I see that we can ship audit logs to Elastic/Splunk for on-prem server. Can this also be done for Teleport Cloud? Is there a timeline for this being available?

Thanks,
Gord T.

**Comment by webvictim** [comment, marked as answer]
The Teleport event handler plugin works with Teleport Cloud as well as on-prem setups. You can use this to import logs from Teleport Cloud and push them to whatever fluentd-supported destination you like.

Docs here: https://goteleport.com/docs/management/export-audit-events/fluentd/
